api.js

```diff
@@ -224,3 +236,86 @@ Api.prototype._run = function (files, _options) {
 		return runStatus;
 	});
 };
+
+Api._blankResults = {
```

Comment:
Everywhere that returns this object structure (but with real data) forms it themselves. On error we need to return an empty object so the `runStatus.processResults(results);` doesn't choke on an undefined so ideally no consumer gets the blank results structure. I only pulled it into its own variable because there are 3 failure cases that need to send it up stream. 

Writing this I realized I can simplify down to 2 and will be pushing a change to simplify as well as changing it to a getter Just In Case™